dopamine/discrete_domains/run_experiment.py

```diff
@@ -608,10 +608,12 @@ def _run_iterations(self):
     # attribute.
     self._completed_iteration = self._start_iteration
     for iteration in range(self._start_iteration, self._num_iterations):
-      if (iteration + 1) % self._eval_period == 0:
+      if iteration and iteration % self._eval_period == 0:
```

Comment:
This isn't exactly the right time or place to comment, but do you think `self._eval_period` is a good choice for an argument to the `AsyncRunner` constructor?

Reply:
I think it is a good choice. We can definitely add it.

Reply:
I added a TODO for that.